src/internal/operators/auditTime.ts

```diff
@@ -22,7 +22,7 @@ import { MonoTypeOperatorFunction, SchedulerLike } from '../types';
  * the time unit determined internally by the optional `scheduler`) has passed,
  * the timer is disabled, then the most recent source value is emitted on the
  * output Observable, and this process repeats for the next source value.
- * Optionally takes a {@link IScheduler} for managing timers.
```

Comment:
These can change to `SchedulerLike`, which is a public interface.

Reply:
(there's more throughout this commit where `Scheduler` could be `SchedulerLike`)

Reply:
I replaced all occurrences where I thought it makes sense. Sometimes it looked more like the text is referring to arguments so I left it with just `scheduler`.